steps/nvm/1.1.0/step.yml

```diff
@@ -0,0 +1,44 @@
+title: Node Version Manager (NVM)
+summary: Install NVM and select Node version
+description: |
+  This step will install a specific node.js version using NVM, and will make it available for all your other steps.
+
+  NVM (Node Version Manager) is a simple bash script to manage multiple active node.js versions.
+  See [here](https://github.com/creationix/nvm) for more details.
+website: https://github.com/almouro/bitrise-nvm-step
+source_code_url: https://github.com/almouro/bitrise-nvm-step
+support_url: https://github.com/almouro/bitrise-nvm-step/issues
+published_at: 2017-11-26T18:10:38.375978913+01:00
+source:
+  git: https://github.com/almouro/bitrise-nvm-step.git
+  commit: 4508f4d25172e0426d2c41b8c49a087efdae3eac
+host_os_tags:
+- osx
+- ubuntu
+type_tags:
+- npm
```

Comment:
Please change these back / normalize according to: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention

It's important for better step search / discoverability :)
Of course if you have any questions just let us know! ;)

Reply:
I sent a PR for this at https://github.com/Almouro/bitrise-nvm-step/pull/4

Reply:
Awesome, thanks!